### PR TITLE
Capitalize function of String.py implemented

### PR DESCRIPTION
**Pull request by Madhav2310**

### CODE REVIEW

**Review by Smit-create — COMMENTED** [review]
Remove the unnecessary changes present in the diff

**Review by Madhav2310 — COMMENTED** [review]
Removed commented code

File: integration_tests/test_string.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from string import (capitalize)
+from ltypes import i32, f64, f32, i64, c32, c64
+
+def test_capitalize():
```
Comment (namannimmo10) [inline integration_tests/test_string.py:4]: Thanks for the PR. You'd need to call this function -- and add this file to `run_tests.py`.

Comment (namannimmo10) [inline integration_tests/test_string.py:4]: Thanks for the PR. You'd need to call this function -- and add this file to `run_tests.py`.

File: src/runtime/string.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+hexdigits: str = digits + 'abcdef' + 'ABCDEF'
+octdigits: str = '01234567'
+punctuation: str = "#$%&'()*+,-./:;<=>?@[\]^_`{|}~"
+
+def capitalize(s: str) -> str:
+    """
+    Return a copy of the string with its first character capitalized and the rest lowercased.
+    """
+    result: str
+    result = s[0].upper() + s[1:]
```
Comment (namannimmo10) [inline src/runtime/string.py:16]: Right now, `.upper()` won't work. We should implement this in the library.

Comment (namannimmo10) [inline src/runtime/string.py:16]: Right now, `.upper()` won't work. We should implement this in the library.

Comment (Madhav2310) [reply src/runtime/string.py:16]: Shall I wait for the implementation of .upper() or proceed via other techniques like adding to its ASCII value to convert to capital?

Comment (certik) [reply src/runtime/string.py:16]: You can implement upper as a function in `string.py` for now.

Comment (faze-geek) [reply src/runtime/string.py:16]: @certik @Madhav2310 I came across this while surfing another `string` issue . I have implemented upper as a function 
```
def upper1(s):        # plain python 
    alphabet = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'
    result = ''
    for x in s:
        for pos in range(52):
            if alphabet[pos] == x:
                i = pos
        if x not in alphabet or i>=26:
            result += x
        else:
            result += alphabet[i+26]
    return result

def upper2(s):      # using some inbuilt functions like ord() , chr()
    result = ''
    for char in s:
        if ord(char) >= 97 and ord(char) <=122 :
            result += chr(ord(char) - 32)
        else :
            result += char
    return result
```
Adding related functions like `lower`,`islower` ,`isupper` wouldn't be difficult either. Should I go ahead with this ? I think these can be updated in `lpython_builtin.py` or simply make a new file `runtime/string.py` which would also support this PR !

Comment (certik) [reply src/runtime/string.py:16]: @faze-geek yes, go ahead and implement those, and add tests.

Comment (Madhav2310) [reply src/runtime/string.py:16]: @faze-geek Thank you for your inputs! I'll go ahead and commit these functions with this PR. Do tell if you come up with further additions. 
cc: @certik @czgdp1807

**Comment by Madhav2310** [comment]
@namannimmo10 @czgdp1807 Could you please assist me with the following error?
```
(lp) madhav@madhav-ThinkPad-E590:~/lpython$ lpython test_string.py
Traceback (most recent call last):
  File "/home/madhav/lpython/src/bin/../runtime/lpython_parser.py", line 6, in <module>
    filename_out = sys.argv[2]
IndexError: list index out of range
The command 'python /home/madhav/lpython/src/bin/../runtime/lpython_parser.py test_string.py' failed.
```

**Comment by Madhav2310** [comment]
Once this is properly implemented, I'll add more tests

**Comment by faze-geek** [comment]
@Madhav2310 have you pushed from your main ? I think it was not in sync with the current master, so you should have fetched and pulled it before . Anyways It is better to open a new branch for a pr. I have tackled this issue previously-

You can restore your master branch like this:
```
# make sure the working directory is clean and you are on master
git status
git checkout master

# completely delete the last x wrongfully pushed commits
git reset --hard HEAD~x

# update local master from the main repo master
git remote add upstream <url of lcompilers/lpython>
git fetch upstream
git rebase upstream/master
```
That will restore your master branch to reflect the main one. To make a PR use
```
# first update your main
git checkout main
git fetch upstream
git rebase upstream/main

# Now create a new branch from your updated main
git checkout -b pr_mybranchname

# Now make changes, commit and push etc.
```

**Comment by Madhav2310** [comment]
Yep, I'll make a new PR for this, thank you!

**Pull request closed without merging**